src/SFA.DAS.RoATPService.Database/Script.PreDeployment1.sql

```diff
@@ -32,5 +32,23 @@ BEGIN
 		UPDATE dbo.Organisations
 		SET OrganisationTypeId = 0
 	END
+END
+
 
-END
\ No newline at end of file
+ -- STORY APR-388  April 2019
+IF NOT EXISTS (SELECT 1 FROM dbo.organisationStatus WHERE id = 3) 
+	BEGIN
+		SET IDENTITY_INSERT dbo.[OrganisationStatus] ON;
+		INSERT INTO [dbo].[OrganisationStatus]
+			   ([ID]
+			   ,[Status]
+			   ,[CreatedAt]
+			   ,[CreatedBy])
+		 VALUES
+			   (3,
+			   'On-boarding'
+			   ,getdate()
+			   ,'System')
+		set identity_insert dbo.[OrganisationStatus] OFF;
+			
+	END
```

Comment:
This script will fail if run on a clean database environment, so I think this should be moved to the post deployment script

Reply:
good point